qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/customers/CustomerDbConnector.groovy

```diff
@@ -28,6 +30,11 @@ class CustomerDbConnector implements CustomerDbGateway{
     this.databaseQueries = databaseQueries
   }
 
+  /**
+   * @inheritDoc
+   * @param criteria
+   * @return
```

Comment:
Do we need the parameter and return values here when we have inheritDoc?
If not I would delete them, otherwise, I would add the description and not leave it empty

Reply:
Same for the other methods in this class

Reply:
To be honest: i don't know

Reply:
I think we could just keep an eye on that and see how the generated doc looks like. We can then update the JavaDoc accordingly in a respective branch